packages/react-scripts/config/paths.js

```diff
@@ -130,26 +130,28 @@ if (
   !reactScriptsLinked &&
   __dirname.indexOf(path.join('packages', 'react-scripts', 'config')) !== -1
 ) {
+  const template = 'template';
```

Comment:
Why was this change made? We'll never use the non-active "template", so we can just revert this.

Reply:
Reverted. Was using to swap between `template`/`template-typescript`, but we can figure this out later in a separate PR if we want to improve it.